service/msteams/ms_teams_test.go

```diff
@@ -0,0 +1,88 @@
+package msteams
+
+import (
+	"context"
+	"testing"
+
+	teams "github.com/atc0005/go-teams-notify/v2"
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNew(t *testing.T) {
+	t.Parallel()
+
+	assert := require.New(t)
+
+	service := New()
+	assert.NotNil(service)
+}
+
+func TestMSTeams_AddReceivers(t *testing.T) {
+	t.Parallel()
+
+	assert := require.New(t)
+
+	service := New()
+	assert.NotNil(service)
+
+	service.AddReceivers("https://outlook.office.com/webhook/...")
+	assert.Equal(1, len(service.webHooks))
+
+	service.AddReceivers("https://outlook.office.com/webhook/...", "https://outlook.office.com/webhook/...")
+	assert.Equal(3, len(service.webHooks))
+
+	hooks := []string{"https://outlook.office.com/webhook/...", "https://outlook.office.com/webhook/..."}
+	service.webHooks = []string{}
+	service.AddReceivers(hooks...)
+
+	diff := cmp.Diff(service.webHooks, hooks)
+	assert.Equal("", diff)
```

Comment:
```suggestion
    assert.Equal(service.webHooks, hooks)
```
I think it's more clear that way.